scripts/utils/fee_calculation_functions.py

```diff
@@ -0,0 +1,105 @@
+from utils.data_model.chain_json_model import Chain
+from utils.useful_functions import create_connection_by_url, deep_search_in_object
+
+WEIGHT_PER_SECOND = 1_000_000_000_000
+WEIGHT_PER_MILLIS = WEIGHT_PER_SECOND / 1000  # 1_000_000_000
+WEIGHT_PER_MICROS = WEIGHT_PER_MILLIS / 1000  # 1_000_000
+WEIGHT_PER_NANOS = WEIGHT_PER_MICROS / 1000  # 1_000
+
+
+def base_tx_per_second(base_weight):
+    fee_per_second = WEIGHT_PER_SECOND // base_weight
+    return fee_per_second
+
+
+def connect_to_node(chain: Chain):
+    for node in chain.nodes:
+        connection = create_connection_by_url(node.get('url'))
+        if (connection):
+            return connection
+
+
+def get_base_weight_from_chain(chain: Chain):
+    connection = connect_to_node(chain)
+    weight = connection.get_constant('System', 'BlockWeights').value
+
+    return weight.get('per_class').get('normal').get('base_extrinsic')
+
+
+def biforst_base_fee(chain: Chain) -> float:
+
+    base_weight = get_base_weight_from_chain(chain)
+
+    base_fee_in_ksm = base_tx_per_second(base_weight) * WEIGHT_PER_MILLIS
+
+    return base_fee_in_ksm
+
+
+def heiko_base_fee(chain) -> float:
+    '''
+    https://github.com/parallel-finance/parallel/blob/b72d6afd263086d89d5256b571e522113ebde59f/runtime/heiko/src/constants.rs#L86
+
+    https://github.com/parallel-finance/parallel/blob/b72d6afd263086d89d5256b571e522113ebde59f/runtime/heiko/src/constants.rs#L19
+    '''
+
+    base_weight = get_base_weight_from_chain(chain)
+
+    hko_per_second = base_tx_per_second(base_weight) * 10_000_000_000 // 10
+
+    base_fee_in_ksm = hko_per_second // 50
+
+    return base_fee_in_ksm
+
+
+def kintsugi_base_fee(chain):
+
+    def base_tx_in_ksm():
+        ksm = 1*10**12
+        return ksm // 50_000
+
+    base_weight = get_base_weight_from_chain(chain)
+
+    base_fee_in_ksm = base_tx_per_second(base_weight) * base_tx_in_ksm()
+
+    return base_fee_in_ksm
+
+
+def karura_base_fee(chain):
+
+    base_weight = get_base_weight_from_chain(chain)
+
+    base_fee_in_kar = base_tx_per_second(base_weight) * WEIGHT_PER_MILLIS
```

Comment:
I would right it manually as in implementation to not depend on the constant that original implementation doesn't depend on (WEIGHT_PER_MILLIS).